cypress/integration/models/CreateModel.DisqualifyingEntities.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+*/
+
+const models = require('../../support/Models')
+const modelPage = require('../../support/components/ModelPage')
+const entities = require('../../support/Entities')
+const actions = require('../../support/Actions')
+const helpers = require('../../support/helpers')
+const memoryTableComponent = require('../../support/components/MemoryTableComponent')
+const scorerModal = require('../../support/components/ScorerModal')
+const trainDialogPage = require('../../support/components/TrainDialogsPage')
+const editDialogModal = require('../../support/components/EditDialogModal')
+
+describe("Create Model for Disqualifying Entities tests", () =>
+{
+  after(() => { cy.VerifyMonitorFinds() })
+  
+  it("Create Model for Disqualifying Entities tests", () =>
+  {
+    models.CreateNewModel(`Model-disq-${helpers.ModelNameTime()}`)
+    
+    entities.CreateNewEntity({name: 'name'})
```

Comment:
Would like to see consts for "name", "want" and "sweets"

Reply:
I had made the changer earlier, but I did not like it because:
1. Did not like how the code read.
2. Did not like what it took to create it.
3. I don't believe the benefit is worth the cost.

However, if you insist, I will check in the attached change...
[CreateModel.DisqualifyingEntities.js.txt](https://github.com/Microsoft/ConversationLearner-UI/files/2494228/CreateModel.DisqualifyingEntities.js.txt)